ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -522,152 +476,132 @@ def check_backends(logger):
+            objects_to_repair = disk_safety_info['objects_to_repair']
+            disk_lost = disk_safety_info['most_disks_lost']
+            total_objects = disk_safety_info['total_objects']
+            # limit to 4 numbers
+            repair_percentage = float('{0:.4f}'.format((float(objects_to_repair) / total_objects) * 100)) if total_objects != 0 else 0
+            if disk_lost == 0:
+                logger.success('Backend {0} has no disks that are lost.'.format(backend_name))
+            else:
+                msg = "Another {0} disks can be lost." if lowest_safety > 0 else "Losing more disks will cause data loss!"
+                logger.failure('Backend {0} has lost {1} disk(s). {2}'.format(backend_name, disk_lost, msg))
 
-            repair_percentage = 0
-            if len(backends_to_be_repaired) == 0:
-                logger.success("Found no backends with disk lost. All data is safe.")
+            msg = 'Backend {0}: {1} out of {2} objects have to be repaired.'.format(backend_name, objects_to_repair, total_objects)
+            if objects_to_repair == 0:
+                logger.success(msg)
             else:
-                logger.failure("Currently found {0} backend(s) with disk lost.".format(len(backends_to_be_repaired)))
-                for backend, disk_lost in backends_to_be_repaired.iteritems():
-                    # limit to 4 numbers
-                    repair_percentage = float("{0:.4f}".format((float(disk_lost['objects']) /
-                                                                float(disk_lost['total_objects'])) * 100))
-                    logger.warning("Backend {0}: {1} out of {2} objects have to be repaired."
-                                   .format(backend, disk_lost['objects'], disk_lost['total_objects']))
-                    logger.warning('Backend {0}: {1}% of the objects have to be repaired'.format(backend,
-                                                                                                 repair_percentage))
+                logger.warning(msg)
+            if objects_no_safety > 0:
+                logger.failure('Backend {0}: {1} out of {2} objects will be beyond repair if another disk fails.'.format(backend_name, objects_no_safety, total_objects))
+            if lost_objects > 0:
+                logger.failure('Backend {0}: {1} out of {2} objects are beyond repair.'.format(backend_name, lost_objects, total_objects))
             # Log if the amount is rising
             cache = CacheHelper.get()
             repair_rising = False
-            if cache is None:
+            if cache is None or cache.get(backend_name, None) is None:
                 # First run of healthcheck
-                logger.success("Object repair will be monitored on incrementations.")
-            elif repair_percentage == 0:
+                logger.success('Object repair for backend_name {0} will be monitored on increments.'.format(backend_name))
```

Comment:
This is the 2nd time you print a success message mentioning no objects need reparation